templates/CRM/CiviAwards/Form/AwardReview.tpl

```diff
@@ -90,13 +90,13 @@
       {* End if error message *}
 
       {* Form fields section Starts *}
-      {foreach from=$elementNames item=elementName}
+      {foreach from=$elementData item=element}
         <div class="form-group {$form_group_class}">
-          <label class="{$form_group_label_class}">{$form.$elementName.label}</label>
+          <label class="{$form_group_label_class}">{$form[$element.name].label}</label>
           {if $isReviewFromSsp}
-            <div class="ssp-form-control-description text-muted"> Lorem ipsum dolor sit. Sit dolor ipsum lorem porem. </div>
+            <div class="ssp-form-control-description text-muted"> {$element.help_post} </div>
           {/if}
-          <div class="{$form_group_field_class}">{$form.$elementName.html}</div>
+          <div class="{$form_group_field_class}">{$form[$element.name].html}</div>
```

Comment:
And `help_post` should be printed here
```
{if $isReviewFromSsp}
<div class="ssp-form-control-description text-muted"> {$element.help_post} </div>
```
{/if}

Reply:
To be honest, I think you can update the template as you deem fit. I didn't even want to update this. I only wanted to expose the variables.

Reply:
But, Isn't this feature part of this PR. Do you think we should have another PR just to add post and prefix. I think that's an overkill. If we have everything, why can't we just add it and be done with it?

Reply:
But, now I think, it's fine to keep it as it. As we don't know as of now which field will be used as post field. I think we can work on this as a part of the FE ticket as this might need discussion and it is not right to block this ticket/PR for that.